wsd/DocumentBroker.cpp

```diff
@@ -4039,17 +4039,23 @@ void DocumentBroker::syncBrowserSettings(const std::string& userId, const std::s
         it.second->updateBrowserSettingsJSON(key, value);
         if (!upload)
```

Comment:
Is the assumption that a user might have multiple sessions (which is true), and we want to `updateBrowserSettingsJSON()` for all sessions, but `uploadBrowserSettingsToWopiHost()` only once (for the current user) ?

If yes, can we please add a comment? If no, can we please add a comment with the correct logic ?